js/ui/appDisplay.js

```diff
@@ -373,7 +373,7 @@ const AllView = new Lang.Class({
 
     _init: function() {
         this.parent({ usePagination: true }, null);
-        this._scrollView = new St.ScrollView({ style_class: 'all-apps',
+        this._scrollView = new St.ScrollView({ style_class: 'all-apps vfade',
```

Comment:
I just mentioned this to fmuellner on IRC, to get his opinion. Perhaps it's not the real bug and just a sympton, but for now I'd keep it here, as it's consistent with what we already did so far too